evm/src/main/java/org/hyperledger/besu/evm/operation/DifficultyOperation.java

```diff
@@ -27,8 +29,12 @@ public DifficultyOperation(final GasCalculator gasCalculator) {
   @Override
   public Operation.OperationResult executeFixedCostOperation(
       final MessageFrame frame, final EVM evm) {
-    frame.pushStackItem(frame.getBlockValues().getDifficultyBytes());
-
+    if (frame.getBlockValues().getDifficultyBytes() == null
```

Comment:
Don't make this operation do two different things.  Make a separate `RandomOperation` and have the hard fork spec drop it in.

Reply:
I wanted to do that, but since they are the same byte, didn't think it was possible. If you're saying that dropping it in after Difficulty will override it, then that is much cleaner. Will do.

Reply:
Ah now I remember. I didn't do that either because this isn't driven by a protocol schedule. This happens when difficulty becomes zero on the merge. See the definition of `TERMINAL_BLOCK` here https://eips.ethereum.org/EIPS/eip-3675#definitions

Reply:
Ok, so what we could do, is drop in RANDOM to overwrite DIFFICULTY for 0x44, at some hardfork block before the merge. The conditional has to stay in RANDOM though, since it shouldn't change behavior till difficulty value disappears from block. Then we clean it up in the first release after the merge.

Reply:
I think this is the most viable option, given the merge transition mechanism.  If an EIP-4488 hardfork does not happen pre-merge, we could add it to the soft fork that is (was?) slated to happen a couple weeks prior to the merge which adds the target terminal total difficulty.